### ISSUE TITLE
Accessibility: Reading Order on Current Tab

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1008]
### Current
Upon loading a location page and navigating the tabs. as you tab through the tabs and the focus within the Current tab. The screen reader goes to the observation station info first and then reads the descriptive forecast, while the focus box remains on the observation link.

![Image](https://github.com/weather-gov/weather.gov/assets/24654605/fd107c15-3edc-4c41-bb62-cfee08fb3f3c)

### Expected
Upon focusing within the Current tab, screen reader should:
1. Read the summarized version of current conditions first
2. Read Observation Station and the link second
3. Read Last update time last. 

### Acceptance criteria
- [ ] Order has been updated to reflect expected behavior
- [ ] Tests have been implemented or modified to capture the bug
- [ ] Changes have been tested for accessibility

**Comment by partly-igor** [comment]
Plain text typically doesn't receive focus (that is the focus outline, not what the screenreader focus) so I am not sure we want to add a focus ring to it. But what we do need to do is make sure that the `sr-only` description we currently have comes before the Observation station content, currently we have that element after the two visual columns. That would match up with the expected reading order. T

**Comment by colinmurphy01** [comment]
@igorkorenfeld please edit the expected behavior above for clarity. I'll default to your recommendations.

**Comment by partly-igor** [comment]
Updated
